apps/wallet/src/ui/app/pages/accounts/ExportAccountPage.tsx

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { fromB64 } from '@mysten/sui.js/utils';
+import { bytesToHex } from '@noble/hashes/utils';
+import { useMutation } from '@tanstack/react-query';
+import { useMemo } from 'react';
+import { Navigate, useNavigate, useParams } from 'react-router-dom';
+
+import { HideShowDisplayBox } from '../../components/HideShowDisplayBox';
+import { VerifyPasswordModal } from '../../components/accounts/VerifyPasswordModal';
+import Alert from '../../components/alert';
+import Loading from '../../components/loading';
+import Overlay from '../../components/overlay';
+import { useAccounts } from '../../hooks/useAccounts';
+import { useBackgroundClient } from '_src/ui/app/hooks/useBackgroundClient';
+
+export function ExportAccountPage() {
+	const { accountID } = useParams();
+	const { data: allAccounts, isLoading } = useAccounts();
+	const account = useMemo(
+		() => allAccounts?.find(({ id }) => accountID === id) || null,
+		[allAccounts, accountID],
+	);
+	const backgroundClient = useBackgroundClient();
+	const exportMutation = useMutation({
+		mutationKey: ['export-account', accountID],
+		mutationFn: async (password: string) => {
+			if (!account) {
+				return null;
+			}
+			return await backgroundClient.exportAccountKeyPair({ password, accountID: account.id });
+		},
+	});
+	const privateKey = useMemo(() => {
```

Comment:
same re: memo here

Reply:
Yeah now that you say it no, probably not necessary. There aren't many cases that this will run and it wont be because of the keypair. I will move it to the mutation so I will remove this anyway!